project/views.py

```diff
@@ -1,23 +1,27 @@
-from flask import render_template, redirect, session, url_for
+"""Views for Tidbits."""
+
+import logging
+
+from flask import redirect, render_template, session, url_for
 
 from project.app import app
 from project.forms import QueryForm
 
-import logging
-
 
 @app.route('/', methods=['GET', 'POST'])
 def home():
+    """Home page."""
     form = QueryForm()
     if form.validate_on_submit():
         session['query_result'] = form.query.data
-        logging.debug('User queried:|{}|'.format(session['query_result']))
```

Comment:
Why not keep as format()?

Reply:
We should prefer letting logging fill in the arguments as opposed to formatting the strings ourselves.

Reply:
oh, ok. didn't notice the different number of parameters there